src/Security/Authentication/Google/src/PACKAGE.md

```diff
@@ -0,0 +1,39 @@
+## About
+
+`Microsoft.AspNetCore.Authentication.Google` contains middleware to support Google's OpenId and OAuth 2.0 authentication workflows.
```

Comment:
I don't think the Google `AuthenticationHandler` supports OIDC. It's an `OAuthHandler` but not an `OpenIdConnectHandler`.

```suggestion
`Microsoft.AspNetCore.Authentication.Google` contains middleware to support Google's OAuth 2.0 authentication workflow.
```

Reply:
That's fair, I'll change that (this line was copied from the `<Description>` in the `.csproj`, maybe that should be changed as well).